console/encrypter.py

```diff
@@ -0,0 +1,105 @@
+from cryptography.hazmat.backends.openssl.backend import backend
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.primitives.serialization import Encoding, PublicFormat
+
+import base64
+import os
+from console import settings
+import jwt
+import hashlib
+import logging
+
+
```

Comment:
We can use the crypt library instead of this class: https://github.com/ONSdigital/sdc-cryptography

You can see an example here:
https://github.com/ONSdigital/sdx-seft-consumer-service/blob/master/app/tests/test_publish_receive.py